### PR TITLE
Refactor static viz to be compatible with individual cards with multiple series

### PR DESCRIPTION
**Pull request by howonlee**
This was a lacking feature which wasn't put in #18676 which nevertheless should go in #18676, noted during the bug bash. This is the other idiosyncratic way to have multiple series in one chart, just shove them in without actually denoting a combo officially. So the combo way, you have an explicit `combo` display_type: this one you just have `bar` or something and want multiple series in there anyhow. Nevertheless, I just use the existing debugged combo implementation.

### CODE REVIEW

**Review by alxnddr — APPROVED** [review]
Checked for all viz types, works great 🏅 

Screenshot 2021-12-17 at 14 09 32

There is one thing we can improve separately — choose Y-axis based on visual settings or based on series data so that if one series has Y-range 0-1000 and the other 0-50 it renders two Y-axes like:
Screenshot 2021-12-17 at 14 16 03

I added it to https://github.com/metabase/metabase/issues/18676

**Comment by howonlee** [comment]
> There is one thing we can improve separately — choose Y-axis based on visual settings or based on series data so that if one series has Y-range 0-1000 and the other 0-50 it renders two Y-axes like: Screenshot 2021-12-17 at 14 16 03
> 
> I added it to #18676

I found the actual-actual default type behavior in a brutally obscure corner of the settings for the non-static viz, so I think I am going to repeat this trick. Unfortunately the result is ever-more coupling between the static viz and the non-static viz but that was probably inevitable

File: src/metabase/pulse/render/body.clj [hunk 1]
```diff
@@ -586,28 +565,71 @@
...
 
-(s/defmethod render :combo :- common/RenderedPulseCard
-  [_ render-type _timezone-id :- (s/maybe s/Str) card _ {:keys [cols rows viz-settings] :as data}]
-  ;; Special axis-rowfns because we have more than 1 axis
-  (let [clean-rows       (common/non-nil-combo-rows rows)
-        x-axis-rowfn     (ui-logic/mult-x-axis-rowfn card (assoc data :rows clean-rows))
-        y-axis-rowfn     (ui-logic/mult-y-axis-rowfn card (assoc data :rows clean-rows))
-        x-rows           (map x-axis-rowfn clean-rows)
-        y-rows           (map y-axis-rowfn clean-rows)
+(defn- lab-image-bundle
```
Comment (dpsutton) [inline src/metabase/pulse/render/body.clj:568]: what does the prefix "lab" mean here?

Comment (dpsutton) [reply src/metabase/pulse/render/body.clj:568]: Also, this docstring reads more like a comment. If i got that back as a docstring it wouldn't help me I don't think. Both of these are minor nits but in 3 months I won't understand either.

**Review by dpsutton — APPROVED** [review]

**Pull request merged**